sympy/solvers/ode/tests/ode_test_suite/test_kamke.py

```diff
@@ -0,0 +1,413 @@
+        "kamke_1.291": a*(alpha*x + bbeta*y(x))**2 - alpha*(a*x + b*y(x)) + (b*(alpha*x + bbeta*y(x))**2 - bbeta*(a*x + b*y(x)))*Derivative(y(x), x),
+        "kamke_1.292": (a*y(x) + b*x + c)**2*Derivative(y(x), x) + (alpha*y(x) + bbeta*x + ggamma)**2,
+        "kamke_1.293": x*(-3*x + y(x)**2)*Derivative(y(x), x) - 5*x*y(x) + 2*y(x)**3,
+        "kamke_1.294": x*(-a + x**2 + y(x)**2)*Derivative(y(x), x) - (a + x**2 + y(x)**2)*y(x),
+        "kamke_1.295": x**2*y(x) + x*(-x**2 + x*y(x) + y(x)**2)*Derivative(y(x), x) + x*y(x)**2 - y(x)**3,
+        "kamke_1.296": x**4 - 2*x**2*y(x)**2 + x*(x**2*y(x) + x**2 + y(x)**2)*Derivative(y(x), x) - 2*y(x)**3,
+        "kamke_1.297": -x**2*y(x) + 2*x*(5*x**2 + y(x)**2)*Derivative(y(x), x) + y(x)**3,
+        "kamke_1.298": 3*x*y(x)**2*Derivative(y(x), x) - 2*x + y(x)**3,
+        "kamke_1.299": -2*x*y(x) + (-x**2 + 3*x*y(x)**2)*Derivative(y(x), x) + y(x)**3,
+        "kamke_1.300": 6*x*y(x)**2*Derivative(y(x), x) + x + 2*y(x)**3
+    }
+
+    def get_example(self, example):
+        return self.eqs[example]
+
+    def test_example(self, example, hint="default", verify=True, dsolve_timelim=10, checkodesol_timelim=10, log=False):
+        start = time.time()
+        eq = self.get_example(example)
+        sol = None
+        output = example + " "
+
+        try:
+            # Try to find the solution to the equation
+            with time_limit(dsolve_timelim):
+                sol = dsolve(eq, y(x), hint)
+                if isinstance(sol, Eq):
+                    sol = [sol]
+        except (TimeOutError, ValueError, NotImplementedError, TypeError):
+            # Solution not found / timeout
+            output += "dsolve failed"
+
+        # If a solution is found
+        if sol is not None and verify:
+            try:
+                # Try to verify if the solution is correct
+                assert len(sol)
+                with time_limit(checkodesol_timelim):
+                    checks = checkodesol(eq, sol, y(x))
+                assert any([x[0] for x in checks])
+                output += "solved"
+            except AssertionError:
+                # Wrong solution
+                output += "dsolve failed"
+            except (TimeOutError, ValueError, NotImplementedError, TypeError):
+                # Checkodesol unable to verify / timeout
+                output += "checkodesol failed"
+        output += f" in {time.time() - start} seconds\n"
+        returns = sol
+        if log:
+            returns = [sol, output]
+        return returns
+
+
+    def test_all_examples(self, hint="default", verify=True, dsolve_timelim=10, checkodesol_timelim=10, report_path=None):
+        if report_path is None:
+            report_path = "test_report.txt"
+
+        start = time.time()
+        # Counts for result
+        solved = 0
+        check_slow = 0
+        fail = 0
+
+        with open(report_path, "w") as output_file:
```

Comment:
I think it would be better to just print the output. If you want it in a file, you can redirect the terminal output.